packages/README.md

```diff
@@ -0,0 +1,192 @@
+		"dependencies": {
+			"@babel/runtime": "^7.8.3"
+		},
+		"publishConfig": {
+			"access": "public"
+		}
+	}
+	```
+	This assumes that your code is located in the `src` folder and will be transpiled with `Babel`.
+2. `.npmrc` file which disables creating `package-lock.json` file for the package:
+	```
+	package-lock=false
+	```
+3. `README.md` file containing at least:
+	- Package name
+	- Package description
+	- Installation details
+	- Usage example
+	- `HURUmap` logo (`<br/><br/><p align="center"><img src="https://hurumap.org/static/img/logo-white.png" alt="HURUmap" /></p>`)
+
+## Managing Dependencies
+
+There are two types of dependencies that you might want to add to one of the existing HURUmap packages.
+
+### Production Dependencies
+
+Production dependencies are stored in the `dependencies` section of the package’s `package.json` file.
+
+#### Adding New Dependencies
+ 
+The simplest way to add a production dependency to one of the packages is to run a very convenient [lerna add](https://github.com/lerna/lerna/tree/master/commands/add#readme) command from the root of the project.
+
+_Example:_
+
+```bash
+lerna add lodash packages/charts
+```
+
+This command adds the latest version of `lodash` as a dependency to the `@hurumap/charts` package, which is located in `packages/charts` folder.
+
+#### Removing Existing Dependencies
+
+Removing a dependency from one of the HURUmap packages requires some manual work. You need to remove the line in the corresponding `dependencies` section of the `package.json` file.
+
+_Example:_
+
+```diff
++++ b/packages/scripts/package.json
+@@ -43,7 +43,6 @@
+                "check-node-version": "^3.1.1",
+                "cross-spawn": "^5.1.0",
+                "eslint": "^5.16.0",
+-               "jest": "^24.7.1",
+                "jest-puppeteer": "^4.0.0",
+                "minimist": "^1.2.0",
+                "npm-package-json-lint": "^3.6.0",
+```
+
+Next, you need to run `npm install` in the root of the project to ensure that `package-lock.json` file gets properly regenerated.
+
+#### Updating Existing Dependencies
+
+This is the most confusing part of working with [lerna] which causes a lot of hassles for contributors. The most successful strategy so far is to do the following:
+ 1. First, remove the existing dependency as described in the previous section.
+ 2. Next, add the same dependency back as described in the first section of this chapter. This time it wil get the latest version applied unless you enforce a different version explicitly.
+ 
+### Development Dependencies
+
+In contrast to production dependencies, development dependencies shouldn't be stored in individual WordPress packages. Instead they should be installed in the project's `package.json` file using the usual `npm install` command. In effect, all development tools are configured to work with every package at the same time to ensure they share the same characteristics and integrate correctly with each other.
```

Comment:
WordPress?

Reply:
I'll replace that, drew inspiration from wordpress.